websrc/control_test/src/control_test_main.tsx

```diff
@@ -76,7 +76,7 @@ class ControlTest extends React.Component< {}, ControlTestState >
 								<div className="SliderControl">
 									<AvPanelAnchor>
 										<AvSlider rangeX={ 0.7 } onSetValue={ this.onSetSlider }
-											modelUri={ g_builtinModelGear }/>
+											modelUri={ g_builtinModelAardvark }/>
```

Comment:
The aardvark seems like a strange choice for a slider. How about replacing the big ball at the bottom instead?

Reply:
Sounds good!